src/contracts/interfaces/IEigenPodManager.sol

```diff
@@ -95,34 +148,18 @@ interface IEigenPodManager is IPausable {
      * Likewise, when a withdrawal is completed, this "deficit" is decreased and the withdrawal amount is decreased; We can think of this
      * as the withdrawal "paying off the deficit".
      */
-    function podOwnerShares(address podOwner) external view returns (int256);
```

Comment:
This function name is still mentioned in `docs/core/EigenPod.md` and `docs/core/EigenPodManager.md` [[ref](https://github.com/Layr-Labs/eigenlayer-contracts/blob/slashing-magnitudes/docs/core/EigenPodManager.md#important-state-variables)]

Reply:
/docs are still a WIP. Will address this